libs/api/domains/license-service/src/lib/client/machine-license-client/machineLicenseMappers.ts

```diff
@@ -0,0 +1,74 @@
+import { VinnuvelaDto } from '@island.is/clients/adr-and-machine-license'
+import {
+  GenericLicenseDataFieldType,
+  GenericUserLicensePayload,
+} from '../../licenceService.type'
+
+export const parseMachineLicensePayload = (
+  license: VinnuvelaDto,
+): GenericUserLicensePayload | null => {
+  if (!license) return null
+  const data = [
+    {
+      type: GenericLicenseDataFieldType.Value,
```

Comment:
sync labels
https://www.figma.com/file/kzvPtwIM25MrBmoKOhEkji/%C3%8Dsland.is-%E2%80%93-App?node-id=2747%3A27519